graphrag/index/flows/create_final_text_units.py

```diff
@@ -42,16 +42,16 @@ async def create_final_text_units(
     aggregated = final_joined.groupby("id", sort=False).agg("first").reset_index()
 
     is_using_vector_store = False
-    if text_embed:
-        aggregated["text_embedding"] = await text_embed_df(
+    if text_text_embed:
```

Comment:
wonder why the extra text_?

Reply:
Because the naming convention on each of these is {field}_text_embed

Reply:
Sound like moon moon haha, but approved